src/js/themes/base.js

```diff
@@ -1663,6 +1682,11 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         size: 'medium',
       },
     },
+    starRating: {
```

Comment:
I can strip these out just left them incase we want to see story and change naming

Reply:
I'm inclined to not have this color here as part of the base theme